tensorflow/core/util/ctc/ctc_beam_search.h

```diff
@@ -196,7 +199,7 @@ Status CTCBeamSearchDecoder<CTCBeamState, CTCBeamComparer>::Decode(
     }
 
     Status status =
-        TopPaths(top_n, &beams, &beam_log_probabilities, merge_repeated_);
+        TopPaths(top_n, &beams, &beam_log_probabilities, this->merge_repeated_);
```

Comment:
why?

Reply:
Empirically answer:
/tensorflow/core/util/ctc/ctc_beam_search.h:202:58: error: 'merge_repeated_' was not declared in this scope
         TopPaths(top_n, &beams, &beam_log_probabilities, merge_repeated_);
                                                          ^~~~~~~~~~~~~~~
I don't know the exact standard paragraph.